tier1/checklist.md

```diff
@@ -0,0 +1,371 @@
+# DSAC OSPO Outbound Review Checklist
+## Tier 1: One-Time Release
+
+### Instructions
+
+This is a review process to approve CMS-developed software to be released open source. If you would like your repository to be released, please complete the following steps.
+
+[Instructions](#Instructions)
+
+[State the Benefit(s) of Open Sourcing the Project
```

Comment:
Link is not working here (I see the other links are highlighted but this one isn't)